ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -49,3 +49,11 @@ def eq(input, other, *, out=None):
 
 def argsort(input, dim=-1, descending=False):
     return ivy.argsort(input, axis=dim, descending=descending)
+
+
+def sort(input, dim=-1, descending=False, stable=False, out=None):
+    values = ivy.sort(input, axis=dim, descending=descending, stable=stable, out=out)
+
+    indices = ivy.argsort(input, axis=dim, descending=descending)
+
+    return [values, indices]  # should be a namedtuple?
```

Comment:
According to the torch documentation

> A namedtuple of (values, indices) is returned, where the values are the sorted values and indices are the indices of the elements in the original input tensor.

So i guess it would be better to return a named tuple.

Reply:
Agreed. Looking at https://pytorch.org/docs/stable/jit_language_reference.html, it looks like torch uses `collections.namedtuple`, so I've updated sort frontend to match.